src/common/enclave_interface_types.h

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+enum CreateNodeStatus
+{
+  OK = 0,
+  InternalError = 1,
+  NodeAlreadyCreated = 2,
+  VersionMismatch = 3,
+  ConsensusNotAllowed = 4,
+  TooManyThreads = 5,
```

Comment:
MoreThreadsThanTCSCount would be a little more precise here

Reply:
This seems to be hardcoded to `24` in our code base, independently of the `oe_sign.conf` `NumTCS`. Am I missing something?